src/NetworkName.php

```diff
@@ -475,25 +475,31 @@ public static function showFormForNetworkPort($networkPortID)
         );
         echo "</td>\n";
 
-        echo "</tr><tr class='tab_bg_1'>\n";
-
-        echo "<td>" . IPAddress::getTypeName(Session::getPluralNumber());
-        IPAddress::showAddChildButtonForItemForm($name, 'NetworkName__ipaddresses');
-        echo "</td>";
-        echo "<td>";
-        IPAddress::showChildsForItemForm($name, 'NetworkName__ipaddresses');
-        echo "</td>";
-
-       // MoYo : really need to display it here ?
-       // make confure because not updatable
-       // echo "<td>".IPNetwork::getTypeName(Session::getPluralNumber())."&nbsp;";
-       // Html::showToolTip(__('IP network is not included in the database. However, you can see current available networks.'));
-       // echo "</td><td>";
-       // IPNetwork::showIPNetworkProperties($name->getEntityID());
-       // echo "</td>\n";
-        echo "<td colspan='2'>&nbsp;</td>";
-
-        echo "</tr>\n";
+        echo "</tr>";
+
+        if ($name->isNewItem()) {
+            if (!$name->canCreate()) {
+                $canedit = false;
+            }
+            $canedit = $name->canUpdate();
```

Comment:
Why checking for update rigth when adding? (I'ma always a bit lost in glpi acls, but that sounds strange)